<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test with firefox"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="94.415">
  <properties />
            <testcase name="TC03_VALIDATE_POST_CONTENT" time="94.415" classname="testcases.newsfeed.NewsfeedTab.PostFunction.NormalPost.TestcaseOfPost">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  java.base/java.util.Objects.requireNonNull(Objects.java:222)
  org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:85)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
  actions.common.Function.AbstractPage.waitElementToClickAble(AbstractPage.java:387)
  actions.PageObject.Newsfeed.PageFeed.PostFunction.EditorFunctionObject.clickToSharePostButton(EditorFunctionObject.java:109)
  testcases.newsfeed.NewsfeedTab.PostFunction.NormalPost.TestcaseOfPost.TC03_VALIDATE_POST_CONTENT(TestcaseOfPost.java:106)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:564)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
